pyvista/plotting/renderer.py

```diff
@@ -2891,6 +2891,13 @@ def close(self):
             self._empty_str.SetReferenceCount(0)
             self._empty_str = None
 
+    def update(self):
+        """Update the renderer."""
+        if self.__charts is not None:
+            for chart in self.__charts:
+                # Notify ChartMPLs to redraw themselves when plotter.update() is called
+                chart._render_event(plotter_update=True)
```

Comment:
There isn't an `Update()` method on the `vtkRenderer` class, so I'd like to avoid adding this method here as it introduces an inconsistency with VTK. "update" calls are typically for pipelining whereas, in the rendering flow, the "render" calls trigger updates.

Should this logic be in a new `render()` method instead, since it is forcing a render event? Which would also call `self.Render()` on the `vtkRenderer`?

Reply:
I suppose this gets nuanced/confusing with the `update()` method on the `BasePlotter`...

Reply:
Renamed and moved everything to `render` calls.